scripts/world_bank/worldbank_ids/requirements.txt

```diff
@@ -0,0 +1 @@
+bblocks
```

Comment:
Please include with version like bblocks==1.4.0

Reply:
Included the version.